### PR TITLE
Add new Testing Tools

### PR DESCRIPTION
**Pull request by Hiro-Nakamura**
Cypress cy.commands to support the new `VersionCheck()` and `TestLog()` features.

Note: this depends on 
- [Create a default version check handler #67](https://github.com/digi-serve/ab-utils/pull/67)
- [Add new Testing Tools #144](https://github.com/digi-serve/ab_service_api_sails/pull/144)

## Release Notes

- [wip] Add new versionCheck and testLog features

### CODE REVIEW

**Review by guyyoo — APPROVED** [review]
LGTM!

File: test/e2e/cypress/support/commands.js [hunk 1]
```diff
@@ -155,3 +155,13 @@ Cypress.Commands.add("RunSQL", (folder, files, fail = true) => {
...
    });
 });
+
+Cypress.Commands.add("TestLog", (log) => {
+   // Have the running services clear their definitions.
+   cy.request("POST", "/testlog", { log });
+});
+
+Cypress.Commands.add("VersionCheck", () => {
+   // Have the running services clear their definitions.
```
Comment (nh758) [inline test/e2e/cypress/support/commands.js:165]: maybe, update the comment

**Review by nh758 — CHANGES_REQUESTED** [review]
I think this would be better as part of a global `beforeEach`. At least the `TestLog`.

```js
 beforeEach(() => {
    cy.request("POST", "/testlog", { log: Cypress.currentTest.title });
  });
  ```
  
  Unless we don't want it for each test?
  
  We can add a global `beforeEach` in a [support file](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file) (`cypress/support/e2e.js`)

File: test/e2e/cypress/support/commands.js (hunk 1 above)
Comment (Hiro-Nakamura) [reply test/e2e/cypress/support/commands.js]: OK, yeah that is nice.  I was looking for something like `Cypress.currentTest.title` but didn't find it.  

I'm making that change

**Pull request merged**